src/main/java/org/spoken_tutorial/health/elasticsearch/JsonService/JsonService.java

```diff
@@ -83,4 +84,165 @@ public String saveNarrationToFile(String url, String documentId) throws ParseExc
         }
         return document;
     }
+
+    public String saveNarrationAndCuefScriptoHtmlFile(String url, int tutorialId, int lanId)
+            throws ParseException, IOException {
+
+        String jsonString = "";
+        boolean flag = true;
+        jsonString = restTemplate.getForObject(url, String.class);
```

Comment:
Move this line inside brackets.

Reply:
These lines cant be moved into brackets , because jsonString is used by both if and else.

Reply:
Move this line to else.

It is already being assigned in if.